lib/src/Pos/Context/Context.hs

```diff
@@ -18,6 +18,9 @@ module Pos.Context.Context
        , BlockRetrievalQueue
 
        , ConnectedPeers(..)
+
+       -- * Lenses
+       , ncLastKnownHeader_L
```

Comment:
No need to export this. Just use `lensOf`, following the same pattern as the other `HasLens` instances.

Reply:
For some reason it didn't work out of the box. Let me try.